database/postgres/seed.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+const args = require('minimist')(process.argv.slice(2));
+const fastcsv = require('fast-csv');
+const db = require('./index');
+const contains = require('validator/lib/contains');
+
+const insertFromCsv = () => {
+  let csvData = [];
+  return (
+    fastcsv
+      .parse()
+      .validate((data) => !contains(data[0], ','))
+      .on('data', (data) => {
+          csvData.push(data);
+      })
+      .on('data-invalid', (row, rowNumber) =>
+          console.log(
+              `Invalid [rowNumber=${rowNumber}] [row=${JSON.stringify(row)}]`
+          )
+      )
+      .on('end', () => {
+          const query =
```

Comment:
not sure if you're doing this. The COPY method can be used to bulk insert data into postgres, as long as your csv file exactly matches your table schema !